### ISSUE TITLE
[T2] Grafo Neo4j

### ISSUE THREAD

**Issue by pedrozavalat** [IIC2440/Syllabus-2024-1#24]
Hola! Tengo una duda con la primera parte de la tarea,
Para la aplicacion del algoritmo con algun grafo en Neo4j deberiamos utilizar el grafo entregado de la ultima actividad formativa?
Muchas gracias!

**Comment by juanreutter** [comment]
No, no vamos a subir ninguno para testear, pueden usar el de clases!
